### ISSUE TITLE
block reward

### ISSUE THREAD

**Issue by Gurenduben** [iceland2k14/solominer#5]
if i find a block, how will it get to my btc address? or do i change the address in solo_miner.py?

**Comment by iceland2k14** [comment]
Yes off course. Use your address otherwise the mining block reward will come to my address

**Comment by Gurenduben** [comment]
also how can i set it to run automatically when its finished because i dont want to be manually running the script everytime

**Comment by Newinvestcoin** [comment]
> кроме того, как я могу настроить его на автоматический запуск, когда он закончен, потому что я не хочу каждый раз запускать скрипт вручную
you can get this and rename solo_miner.txt to solo_miner.py, change you address and run
[solo_miner.txt](https://github.com/iceland2k14/solominer/files/6934071/solo_miner.txt)

**Comment by evgenyunholy** [comment]
> > кроме того, как я могу настроить его на автоматический запуск, когда он закончен, потому что я не хочу каждый раз запускать скрипт вручную
> > you can get this and rename solo_miner.txt to solo_miner.py, change you address and run
> > [solo_miner.txt](https://github.com/iceland2k14/solominer/files/6934071/solo_miner.txt)

My fork. Solo mining in python https://github.com/evgenyunholy/solominer
